startupscript/post-startup.sh

```diff
@@ -97,7 +97,7 @@ ${RUN_AS_USER} "mv '${JAVA_DIRNAME}' '${USER_HOME_LOCAL_SHARE}'"
 
 # Create a soft link in /usr/bin to the java runtime
 ln -sf "${USER_HOME_LOCAL_SHARE}/${JAVA_DIRNAME}/bin/java" "/usr/bin"
-chown --no-dereference ${user}:${user} "/usr/bin/java"
```

Comment:
the group that jovyan is in is users, not jovyan. 

i double checked the change for rstudio and vscode